examples/datasets.yml

```diff
@@ -95,3 +95,9 @@ data:
     files:
       - NSRDB_StationsMeta.csv
 
+  - url: http://s3.amazonaws.com/datashader-data/edge-bundling.snappy.parq.zip
+    title: 'Graph for Edge Bundling'
+    files:
+      - nodes.snappy.parq
+      - edges.snappy.parq
```

Comment:
The dataset should be renamed to match what it actually contains, e.g. 'calvert_uk_research2017_nodes.snappy.parq' and 'calvert_uk_research2017_edges.snappy.parq'.